dev-lang/jimtcl/jimtcl-0.83.ebuild

```diff
@@ -1,40 +1,49 @@
 # Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="8"
+EAPI=8
+
+inherit toolchain-funcs edo
 
 DESCRIPTION="Small footprint implementation of Tcl programming language"
 HOMEPAGE="https://jim.tcl.tk/"
 SRC_URI="https://github.com/msteveb/jimtcl/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="LGPL-2"
 SLOT="0/${PV}"
-KEYWORDS="amd64 arm ~arm64 ~m68k ~mips ~s390 x86"
+KEYWORDS="~amd64 ~arm ~arm64 ~m68k ~mips ~s390 ~x86"
 IUSE="doc static-libs"
 
-DEPEND="
-	dev-lang/tcl:0
-"
+DEPEND="dev-lang/tcl:0"
+
+src_prepare() {
+	# Avoid tests that need networking
+	rm tests/ssl.test || die
+	default
+}
 
 src_configure() {
-	CCACHE=none econf --disable-docs --shared
-	if use static-libs ; then
+	# Use the provided bootstrap tclsh, avoiding the script picking a different one
+	edo $(tc-getCC) -o jimsh0 autosetup/jimsh0.c
+
+	export CCACHE=none
+	econf --disable-docs --shared
+	if use static-libs; then
 		# The build does not support doing both simultaneously.
 		mkdir static-libs || die
 		cd static-libs || die
-		CCACHE=none ECONF_SOURCE="${S}" econf --disable-docs
```

Comment:
How did you determine the ccache part can be removed? Or is it github's diff being miselading?